### PR TITLE
Release v4.6.0: set default.xml

### PR DESCRIPTION
**Pull request by hilmarm**

### CODE REVIEW

**Comment by hilmarm** [comment]
I was not entirely sure if I should add the tibit.xml, so I added it in a commit on it's own.

File: default.xml [hunk 1]
```diff
@@ -1,19 +1,17 @@
...
-  <project name="poky" path="poky" remote="yocto" revision="dunfell"/>
-  <!-- open embedded layer -->
-  <project name="meta-openembedded" path="poky/meta-openembedded" remote="oe" revision="dunfell"/>
-  <!-- open source bisdn layers -->
-  <project name="bisdn/bisdn-linux.git" path="poky/build-bisdn-linux" remote="github" revision="master"/>
-  <project name="bisdn/bisdn-onie-additions.git" path="poky/bisdn-onie-additions" remote="github" revision="master"/>
-  <project name="bisdn/meta-ofdpa.git" path="poky/meta-ofdpa" remote="github" revision="master"/>
-  <project name="bisdn/meta-open-network-linux.git" path="poky/meta-open-network-linux" remote="github" revision="master"/>
-  <project name="bisdn/meta-switch.git" path="poky/meta-switch" remote="github" revision="master"/>
+  <remote fetch="https://github.com/" name="github"/>
```
Comment (ideaship) [inline default.xml:3]: The commit changes the order name/fetch to fetch/name and also removes a space. Similar changes have been made on other lines. Formatting changes should be in a separate commit along with some explanation.

File: default.xml [hunk 2]
```diff
@@ -1,19 +1,17 @@
...
+  <remote fetch="ssh://git@gitlab.bisdn.de/" name="gitlab"/>
+  <remote fetch="git://git.openembedded.org/" name="oe"/>
+  <remote fetch="git://git.yoctoproject.org/" name="yocto"/>
+
```
Comment (ideaship) [inline default.xml:7]: The commit removes comments. Why?

**Review by ideaship — CHANGES_REQUESTED** [review]

File: default.xml (hunk 1 above)
Comment (KanjiMonster) [reply default.xml:3]: Because the base file is a (semi) handwritten file, while the one replacing it is an autogenerated one (repo discards all comments). There is no intermediary version with just the changes, or just the formatting.

File: default.xml (hunk 2 above)
Comment (KanjiMonster) [reply default.xml:7]: See above.

File: default.xml [hunk 3]
```diff
@@ -1,19 +1,17 @@
...
+  <project dest-branch="v.4.6.0" name="bisdn/bisdn-linux.git" path="poky/build-bisdn-linux" remote="github" revision="23fa24893d2cff7d8f26e91875bfab6d912a740f" upstream="v4.6.0"/>
```
Comment (KanjiMonster) [inline default.xml:8]: This needs to point to a revision with FEEDURIPREFIX set like we did previously in https://github.com/bisdn/bisdn-linux/pull/43/commits/242b108ba9fa7d9a04047c89f7f5571a0138aa23 for v4.5.0

**Review by KanjiMonster — CHANGES_REQUESTED** [review]

File: default.xml (hunk 1 above)
Comment (ideaship) [reply default.xml:3]: It would be easily possible to create an intermediate version to make things easier for reviewers. Even if no such service is offered, an explanation in the commit message and the PR is a must.

Comment (KanjiMonster) [reply default.xml:3]: I disagree, an explanation in the release process yes, but I don't this needs to be explained every time we do a release.

File: default.xml (hunk 3 above)
Comment (hilmarm) [reply default.xml:8]: Cherry-picked the commit that adds the FEEDURIPREFIX commit

Comment (KanjiMonster) [reply default.xml:8]: This needs to point to the revision of the commit, and the commit cannot be part of this PR, as the forced rebase before merge will change any commit ids.

Comment (hilmarm) [reply default.xml]: now points to a revesion that contains the commit

File: default.xml (hunk 1 above)
Comment (hilmarm) [reply default.xml:3]: Can we continue without this, and I will add it to the documentation process for future releases?

Comment (ideaship) [reply default.xml:3]: I will just assume that I don't understand sufficiently what is going on and that you guys will sort it out.

Comment (hilmarm) [reply default.xml:3]: I added a commit message pointing to the generated .xml file.

Comment (KanjiMonster) [reply default.xml:3]: This isn't a stable link, the contents will change with every image build. This will be confusing when the commit hashes e.g. in a week do not match any more at all.

Comment (hilmarm) [reply default.xml:3]: COmmit message removed

**Review by ideaship — APPROVED** [review]
